uber/config.py

```diff
@@ -12,10 +12,71 @@ def mixin(cls, klass):
 
     def include_plugin_config(self, plugin_config):
         """Plugins call this method to merge their own config into the global c object."""
+
         for attr, val in plugin_config.items():
             if not isinstance(val, dict):
                 setattr(self, attr.upper(), val)
 
+        if 'enums' in plugin_config:
+            self.make_enums(plugin_config['enums'])
+
+        if 'dates' in plugin_config:
+            self.make_dates(plugin_config['dates'])
+
```

Comment:
I think the docstrings explain this, but I wanted plugins to be able to define their own ``[enums]`` and ``[dates]`` sections.